packages/appstore/src/components/modals/modal-manager.tsx

```diff
@@ -28,6 +29,7 @@ type TCurrentList = DetailsOfEachMT5Loginid & {
 
 const ModalManager = () => {
     const store = useStores();
```

Comment:
@nijil-deriv better to comment this instead of removing

Reply:
Remove what @mahdiyeh-deriv? I've only added the `is_wallet_enabled` feature flag here.